scripts/liminal

```diff
@@ -84,9 +84,10 @@ def deploy_liminal_core_internal(clean):
 def docker_compose_command(command_name, args):
     docker_compose_path, project_dir = get_docker_compose_paths()
     concated_args = ' '.join(args)
-    subprocess.call(
+    output = subprocess.run(
```

Comment:
what happens when you do run on the command "up"? does it return?

Reply:
Yes the status code and the stderr.
Concerning that scope there's not much use for the return result of the command up.